### PR TITLE
Added link to set performance threshold for Project Performance Dashboard

### PR DESCRIPTION
**Pull request by rickypchen**
@esoergel @czue Per spec on [Google Doc](https://docs.google.com/spreadsheets/d/1hr5-u9Ni90yhumGJqvROeNvqqn0daIPRVLuzCiIbwb8/edit?ts=5760652c#gid=1814269798), I've added a link to the Project Information page, where performance threshold can be set

### CODE REVIEW

**Comment by esoergel** [comment]
Merge conflicts, but looks good otherwise.

File: corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html [hunk 1]
```diff
@@ -204,7 +204,7 @@
     </tr>
 </table>
 <div class="col-md-6">
-    <h2>{% trans "Active Users Trend" %}</h2>
+    <h2>{% trans "Active Users Trend" %} <small><a href="{% url 'domain_internal_settings' domain %}">{% trans "Set Performance Threshold" %}</a></small></h2>
```
Comment (czue) [inline corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html]: it looks like this is a superuser only view right now, so this link should also probably only show up for superusers. when we "release" the report we should make both public

Comment (rickypchen) [inline corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html]: That sounds good!

**Comment by rickypchen** [comment]
@esoergel resolved merge conflict

File: corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html [hunk 2]
```diff
@@ -153,7 +153,15 @@
...
-        <h2>{% trans "Active Users Trend" %}</h2>
+        <h2>{% trans "Active Users Trend" %}
+            {% if request.user.is_superuser %}
+                <small>
+                    <a href="{% url 'domain_internal_settings' domain %}">
+                    {% trans "Set Performance Threshold" %}
+                    </a>
+                </small>
+            {% endif %}
+        </h2>
```
Comment (esoergel) [inline corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html]: :+1: on the improved code formatting too

**Pull request merged**